codecarbon/cli/main.py

```diff
@@ -2,6 +2,7 @@
 import time
 from pathlib import Path
 from typing import Optional
+from uuid import uuid4
```

Comment:
This import seems to not be used, right ?

Reply:
I imported that to set a random experiment id, since I thought the experiment id might be needed even with no api.
Not sure though, I have to check.